packages/webpack-cli/lib/groups/BasicGroup.js

```diff
@@ -15,6 +15,7 @@ class BasicGroup extends GroupHelper {
                 }
                 return result;
             }, []);
+        this._excuted = false;
```

Comment:
Each `GroupHelper` class can have a `this.strategy` parameter. Check `OutputGroup`.
The strategy tell to `webpack-merge` how to handle specific bits of the configuration:

```js
this.strategy = {
   'output.filename': 'prepend',
   'output.path': 'prepend',
};

```